text-minimessage/src/test/java/net/kyori/adventure/text/minimessage/MiniMessageParserTest.java

```diff
@@ -222,79 +220,79 @@ void testAllColorAliases() {
     final String expected = "<test> random strangerclick here <please> to FEEL it";
-    assertEquals(expected, this.PARSER.stripTokens(input));
+    assertEquals(expected, PARSER.stripTokens(input));
   }
 
   @Test
   void testStripInner() {
     final String input = "<hover:show_text:\"<red>test:TEST\">TEST";
     final String expected = "TEST";
-    assertEquals(expected, this.PARSER.stripTokens(input));
+    assertEquals(expected, PARSER.stripTokens(input));
   }
 
   @Test
   void testStripPlaceholders() {
     final String input = "Hello, <red><name>!";
     final String expected = "Hello, !";
-    assertEquals(expected, this.PARSER.stripTokens(input, PlaceholderResolver.placeholders(Placeholder.miniMessage("name", "you"))));
+    assertEquals(expected, PARSER.stripTokens(input, miniMessage("name", "you")));
   }
 
   @Test
   void testEscapeSimple() {
     final String input = "<yellow>TEST<green> nested</green>Test";
     final String expected = "\\<yellow>TEST\\<green> nested\\</green>Test";
-    assertEquals(expected, this.PARSER.escapeTokens(input));
+    assertEquals(expected, PARSER.escapeTokens(input));
   }
 
   @Test
   void testEscapeComplex() {
     final String input = "<yellow><test> random <bold>stranger</bold><click:run_command:test command><underlined><red>click here</click><blue> to <bold>FEEL</underlined> it";
     final String expected = "\\<yellow><test> random \\<bold>stranger\\</bold>\\<click:run_command:test command>\\<underlined>\\<red>click here\\</click>\\<blue> to \\<bold>FEEL\\</underlined> it";
-    assertEquals(expected, this.PARSER.escapeTokens(input));
+    assertEquals(expected, PARSER.escapeTokens(input));
   }
 
   @Test
   void testEscapeInner() {
     final String input = "<hover:show_text:\"<red>test:TEST\">TEST";
     final String expected = "\\<hover:show_text:\"\\<red>test:TEST\">TEST";
-    assertEquals(expected, this.PARSER.escapeTokens(input));
+    assertEquals(expected, PARSER.escapeTokens(input));
   }
 
   // https://github.com/KyoriPowered/adventure-text-minimessage/issues/169
   @Test
   void testEscapeComplexInner() {
     final String input = "<yellow><test> random <bold>stranger</bold><click:run_command:test command><underlined><red>click here <notToken></click><blue> to <bold>FEEL</underlined> it";
     final String expected = "\\<yellow><test> random \\<bold>stranger\\</bold>\\<click:run_command:test command>\\<underlined>\\<red>click here <notToken>\\</click>\\<blue> to \\<bold>FEEL\\</underlined> it";
-    assertEquals(expected, this.PARSER.escapeTokens(input));
+    assertEquals(expected, PARSER.escapeTokens(input));
   }
 
   @Test
   void testEscapePlaceholders() {
     final String input = "Hello, <red><name>!";
     final String expected = "Hello, \\<red>\\<name>!";
-    assertEquals(expected, this.PARSER.escapeTokens(input, PlaceholderResolver.placeholders(Placeholder.miniMessage("name", "you"))));
```

Comment:
just noticed this here, its not 100% clear, especially when you statically import it, what the methods in Placeholder do.
miniMessagePlaceholder would be kinda long tho and redundant when you do Placeholder.miniMessage, so idk